src/modules/dracut/dracut.conf

```
@@ -8,3 +8,8 @@
 # set a custom name, including the path
 #
 initramfsName: /boot/initramfs-freebsd.img
+
+# Optional: define a list of strings to be passed as arguments to Dracut
+# By default, -f is always included
+options:
+  - "-f"
```

Although it would be OK either way.  I think this would be more consistent with how we treat arrays of options in other conf files.

```suggestion
options: [ "-f" ]
```